woocommerce-gateway-ebanx/gateways/class-wc-ebanx-credit-card-gateway.php

```diff
@@ -440,7 +451,7 @@ public function payment_fields() {
 				'cards' => (array) $cards,
 				'cart_total' => $cart_total,
 				'max_instalments' => $max_instalments,
-				'place_order_enabled' => (isset($this->configs->settings['save_card_data']) && $this->configs->settings['save_card_data'] === 'yes'),
+				'place_order_enabled' => (isset($save_card) && $save_card === 'yes'),
```

Comment:
After `$save_card` becomes safely a boolean we can simplify this to be `'place_order_enabled' => $save_card,`.